src/components/Permissions/DataList.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import Page from 'components/Page'
+import PageTitle from 'components/PageTitle'
+import { useI18n } from 'cozy-ui/transpiled/react'
```

Comment:
You should not import from this "root" directory. You should always use the deeper import you can for cozy-ui (in border to get lighter bundler at the end).

import { useI18n } from 'cozy-ui/transpiled/react/I18n'